schematics/transforms.py

```diff
@@ -189,6 +193,10 @@ def export_loop(cls, instance_or_dict, field_converter,
 
     # Return data if the list contains anything
     if len(data) > 0:
+        if fields_order is not None:
+            for key in fields_order:
+                if key in data:
+                    data.move_to_end(key)
```

Comment:
`move_to_end` is only available in Python 3. Here is port of that in Python 2.7 `https://github.com/twitter/commons/blob/master/src/python/twitter/common/collections/ordereddict.py#L285`. But I won't support having extra dependency. If you can have logic to figure out `Py2k/Py3k`  in helper function and do proper transformation, we're good to go.